docs/MODULES.md

```diff
@@ -157,63 +157,73 @@ To use this module you must have it installed. You can check to see if the file
 ### Additional configuration
 Additional configurations can be made in the file `/etc/yggdrasil.iptunnel.conf`
 
-Section `[iptunnel]`
+Section `[general]`
 
-**IPv6nat**
-Disables masquerading of IPv6 tunnels. Set to `false` when routable addresses are being used across the tunnel. Usually when passing an additional subnet.
+**yggdrasil-interface**
+*Server Only*
 
-Default: *true*
-Values: `true` , `false`
+Defines the yggdrasil interface.
 
-To add advertising of the subnet, add the following prefix to /etc/radvd.conf
-```
-   prefix 20xx:xxxx:xxxx:x::/64
-    {
-        AdvOnLink on;
-        AdvAutonomous on;
-    };
-```
+Default: `ygg0`
+Value: `Yggdrasil interface on the system, usually ygg0 or tun0`
 
-**SUBNET4**
-*Server only*
+**exit-interface**
+*Server Only*
 
-Defines ip addresses to add to route table that will be routed over through yggdrasil. Must match ips in `yggdrasil.iptunnel.server`
+Defines the exit interface that is connected to the internet.
 
-Default: 10.10.0.0/24
-Value: `Any ipv4 address range`
+Default: `eth0`
+Value: `Any interface on the system`
 
-**SUBNET6**
-*Server only*
+Section `[ipv6tunnel]`
 
-Defines ipv6 addresses to add to route table that will be routed over through yggdrasil. Must match ips in `yggdrasil.iptunnel.server`
+**nat**
+Disables masquerading of IPv6 tunnels. Set to `false` when routable addresses are being used across the tunnel. This is used when you route another IPv6 route to be used by the remote node to issue IPv6 addresses to its clients.
 
-Default: fd00::/64
-Value: `Any ipv6 address range`
+Default: `true`
+Values: `true` , `false`
 
-**outint**
-*Server Only*
+To add advertising of this subnet, add a prefix to `/etc/radvd.conf` under the correct interface.  For example if the subnet you are routing has the prefix `20xx:xxxx:xxxx::/64`
+
+```
+interface wlan-ap {
+  AdvSendAdvert on;
+  MaxRtrAdvInterval 30;
+  prefix 20xx:xxxx:xxxx::/64
+  {
+    AdvOnLink on;
+    AdvAutonomous on;
+  }
+};
+```
 
-Defines the interface connected to the internet.
+If you installed hostapd you will have this already configured with a prefix.  Simply add the new prefix under the existing one.
```

Comment:
Do they need a comma or some sort of differentiator?

Reply:
; - added